symfony/config/routes.yaml

```diff
@@ -26,9 +26,6 @@ evaluation_modify:
 dashboard_home:
     path: /dashboard
 
-biobank_home:
```

Comment:
I think we need to have this in the `SymfonyMigrationController` in Silex, as we've got a few places that reference it in the code still.

Found by setting these as my only configured sites:

```
gaBypassGroups:
  - email: hpo-site-a@staging-pmi-ops.org
    name: A
  - email: biospecimen-non-pii@staging.pmi-ops.org
    name: Biobank View
```